core/field_angle.js

```diff
@@ -153,7 +152,8 @@ Blockly.FieldAngle.prototype.showEditor_ = function() {
   this.line_ = Blockly.utils.createSvgElement('line',
       {'x1': Blockly.FieldAngle.HALF,
       'y1': Blockly.FieldAngle.HALF,
-      'class': 'blocklyAngleLine'}, svg);
+      'class': 'blocklyAngleLine',
```

Comment:
Looks like this change was not properly merged: https://github.com/google/blockly/pull/1044/files